runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldTableRow.java

```diff
@@ -65,14 +67,17 @@
 		FLAG_UK = new ImageIcon(ImageUtil.getResourceStreamFromClass(WorldHopperPlugin.class, "flag_uk.png"));
 		FLAG_US = new ImageIcon(ImageUtil.getResourceStreamFromClass(WorldHopperPlugin.class, "flag_us.png"));
 		FLAG_GER = new ImageIcon(ImageUtil.getResourceStreamFromClass(WorldHopperPlugin.class, "flag_ger.png"));
+		ICON_FAVORITE = new ImageIcon(ImageUtil.getResourceStreamFromClass(WorldHopperPlugin.class, "favorite.png"));

```

Comment:
Use the in-game star sprite probably. Or reuse the config panel one and move it to common location

Reply:
This is the ingame star sprite.

Reply:
No, I mean from game. E.g SpriteManager.getSprite

Reply:
Ah, I get you. Any particular reason we haven't done so for the flags? I am imagining preference of style.

Reply:
I think @psikoi said the in-game flags are ugly

Reply:
Fair enough, just had a quick look and I completely agree 😂